Pod/Classes/VOKCoreDataCollectionTypes.h

```diff
@@ -0,0 +1,61 @@
+//
+//  VOKCoreDataCollectionTypes.h
+//  Pods
+//
+//  Created by Carl Hill-Popper on 10/23/15.
+//
+//
+
+#ifndef VOKCoreDataCollectionTypes_h
```

Comment:
Probably not, but it makes being backwards compatible super easy.  If we drop support for Xcode 6, I'd say drop this fallback mechanism too.